### PR TITLE
Hide restore column when completed

### PR DESCRIPTION
**Pull request by torchiaf**
#### PR Checklist
- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:
- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:
- Are backend engineers aware of UI changes?
    - [ ] Yes, the backend owner is:

Related Issue https://github.com/harvester/harvester/issues/6195

### Areas or cases that should be tested

- Test Snapshot restores.
- Test Backups restores.

### CODE REVIEW

File: pkg/harvester/models/kubevirt.io.virtualmachine.js [hunk 1]
```diff
@@ -647,6 +647,10 @@ export default class VirtVm extends HarvesterResource {
     return null;
   }
 
+  get isTerminating() {
+    return this?.metadata?.deletionTimestamp;
```
Comment (a110605) [inline pkg/harvester/models/kubevirt.io.virtualmachine.js:651]: deletionTimestamp is string? nit: better convert to boolean

```
return !!this?.metadata?.deletionTimestamp;
```

**Review by a110605 — APPROVED** [review]
Test locally are good,
- restore a VM backup : show restore column with progress bar before completed, hide the restore column once restore completed :heavy_check_mark: 
- restore a VM snapshot: no restore column shown up :heavy_check_mark: 
- 
Screenshot_20240723_155108

**Review by WebberHuang1118 — APPROVED** [review]
LGTM, thanks.

**Comment by torchiaf** [comment]
@mergify backport release-harvester-v1.3 release-harvester-v1.4

**Pull request merged**